src/mmw/apps/modeling/tasks.py

```diff
@@ -332,13 +332,7 @@ def change_key(modification):
 
 @shared_task
 def run_gwlfe(model_input):
-    # TODO pass real input to model instead of reading it from gms file
-    gms_filename = join(dirname(abspath(__file__)),
-                        'mapshed/data/sample_input.gms')
```

Comment:
Agreed, we should delete that file as part of this PR. But let's not merge until we're at a ready state.